js/library.js

```diff
@@ -33,6 +33,134 @@
     });
 })(wp.media.view);
 
+function createPopup() {
+
+    var overlay = document.createElement('div');
+    overlay.id = 'overlay-visualizer';
+    overlay.style.position = 'fixed';
+    overlay.style.top = '0';
+    overlay.style.left = '0';
+    overlay.style.width = '100%';
+    overlay.style.height = '100%';
+    overlay.style.backgroundColor = 'rgba(0, 0, 0, 0.5)';
+    overlay.style.zIndex = '999';
```

Comment:
For this type of styling, you can just add some class in `library.css`. I do not recommend this way of adding things.